### PR TITLE
Make meta-item API compatible with `LocalInternedString::get` soundness fix

### PR DESCRIPTION
**Pull request by petrochenkov**
r? @Zoxc

### CODE REVIEW

File: src/libsyntax/attr/builtin.rs [hunk 1]
```diff
@@ -252,7 +252,7 @@ fn find_stability_generic<'a, I>(sess: &ParseSess,
                 }
             }
 
-            match meta.ident_str().expect("not a stability level") {
```
Comment (Zoxc) [inline src/libsyntax/attr/builtin.rs]: Did you mean to remove `expect` call here?

**Comment by Zoxc** [comment]
The rest of the changes look good to me.

File: src/libsyntax/attr/builtin.rs (hunk 1 above)
Comment (petrochenkov) [reply src/libsyntax/attr/builtin.rs]: Yes, the impossible case will now return an empty string and hit the `unreachable!()` arm in the `match`.

**Comment by mati865** [comment]
Smells like broken Clippy (again) 😄

**Pull request merged**